HW.06.TASK2/Motorcycle.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using static System.DateTime;
+
+namespace HW._06.TASK2
+{
+    public class Motorcycle
+    {
+
+        private DateTime year { get; set; }
+        private string make { get; set; }
+        private Guid vinNumber { get; set; }
+        private string model { get; set; }
+        private int odometer { get; set; }
+   
+        
+        internal Motorcycle GetYear()
+        {
+            Motorcycle motorcycle = new Motorcycle();
+            motorcycle.year = Now;
+            return motorcycle;
+        }
+
+        internal Motorcycle GetModel(string model)
+        {
+            Motorcycle motorcycle = new Motorcycle();
+            motorcycle.model = model;
+            return motorcycle;
+        }
+        
+        internal Motorcycle GetMake(string make)
+        {
+            Motorcycle motorcycle = new Motorcycle();
+            motorcycle.make = make;
+            return motorcycle;
+        }
+        
+        internal Guid GetVinNumber(Guid vin)
+        {
+            Motorcycle motorcycle = new Motorcycle();
+            motorcycle.vinNumber = vin;
+            return vin;
+        }
+        
+        internal Motorcycle GetOdometer(int miles)
+        {
+            Motorcycle motorcycle = new Motorcycle();
+            motorcycle.odometer = miles;
+            if (miles > 100)
+            { 
+                Console.WriteLine("You motorcycle is not valid as odometer is over the allowed limit");
+            }
+            return motorcycle;
```

Comment:
bad design. you do not need to create a new moto object everytime you call this method